package.json

```diff
@@ -45,5 +45,8 @@
 		"vitest": "^2.0.5",
 		"vue-eslint-parser": "^10.1.1"
 	},
-	"packageManager": "pnpm@10.6.4+sha512.da3d715bfd22a9a105e6e8088cfc7826699332ded60c423b14ec613a185f1602206702ff0fe4c438cb15c979081ce4cb02568e364b15174503a63c7a8e2a5f6c"
+	"packageManager": "pnpm@10.6.4+sha512.da3d715bfd22a9a105e6e8088cfc7826699332ded60c423b14ec613a185f1602206702ff0fe4c438cb15c979081ce4cb02568e364b15174503a63c7a8e2a5f6c",
+	"dependencies": {
+		"egoroof-blowfish": "^4.0.1"
```

Comment:
Can you move this to the deemix package's package.json instead of at the root-level?